### PR TITLE
Publish issue and to make dev updated

### PR DESCRIPTION
**Pull request by siddharthashw**
Fixes - #14 

Publish issue and to make dev updated

### CODE REVIEW

**Comment by siddharthashw** [comment]
Don't accept this request for now.

File: Promact.OAuth.Client/src/Promact.OAuth.Client/Promact.OAuth.Client.nuspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>Promact.OAuth.Client</id>
     <version>$version$</version>
-    <title>Promact.OAuth.Client</title>
+    <title>Promact-OAuth-SDK</title>
```
Comment (chintans) [inline Promact.OAuth.Client/src/Promact.OAuth.Client/Promact.OAuth.Client.nuspec]: Why have you changed name?

**Review by chintans — CHANGES_REQUESTED** [review]

File: Promact.OAuth.Client/src/Promact.OAuth.Client/Promact.OAuth.Client.nuspec (hunk 1 above)
Comment (siddharthashw) [reply Promact.OAuth.Client/src/Promact.OAuth.Client/Promact.OAuth.Client.nuspec]: When you will add the environment variable of travis.  I will updated it and then will ask you to accept the request.

I changed the name for publishing it in nuget from my account

Comment (chintans) [reply Promact.OAuth.Client/src/Promact.OAuth.Client/Promact.OAuth.Client.nuspec]: I have already added environment variable

Comment (siddharthashw) [reply Promact.OAuth.Client/src/Promact.OAuth.Client/Promact.OAuth.Client.nuspec]: Now its updated. You can accept the PR

**Review by chintans — APPROVED** [review]

**Pull request merged**